keepassxc-browser/background/client.js

```diff
@@ -59,7 +59,7 @@ const messageBuffer = {
 
     matchAndRemove(msg) {
         for (let i = 0; i < this.buffer.length; ++i) {
-            if (msg.nonce && msg.nonce === keepassClient.incrementedNonce(this.buffer[i].nonce)) {
+            if (msg?.nonce === keepassClient.incrementedNonce(this.buffer[i].nonce)) {
```

Comment:
I think doing this would produce incorrect results. The `?.` operator returns `undefined` if nonce is nullish. So you could potentially compare undefined with undefined, which would evaluate to true.

Not saying that would happen in this instance, but if we blanket apply this operator everywhere it might be possible.

Reply:
True. We should keep the previous implementation just in case. This is the reason I didn't put this in every possible place.